scripts/shell_completions/bash/borg

```diff
@@ -67,6 +67,11 @@ _borg()
             COMPREPLY=( $(compgen -W "${recompress_when}" -- ${cur}) )
             return 0
             ;;
+        '--upgrader')
+            local upgraders="From12To20"
```

Comment:
there is also a no-op "upgrader" (people can use this to transfer archives between related borg2 repos).

Reply:
I honestly would have skipped it, because it is the default. Who would be explicit about their transfer not being an upgrade? I think you'd only add `--upgrader` if you actually want to use something other than the NoOp one.

Then this was not evident from the documentation. It only lists the `From12To20` upgrader in examples and the `NoOp` upgrader is even less visible. I think this should be improved somehow.

In any case, I adapted it as requested.